client/keys/utils.go

```diff
@@ -57,8 +58,7 @@ func printKeyringRecords(w io.Writer, records []*cryptokeyring.Record, output st
 
 	case OutputFormatJSON:
 		// TODO https://github.com/cosmos/cosmos-sdk/issues/8046
-		// Replace AminoCdc with Proto JSON
```

Comment:
I am unaware we have to specifically use Proto JSON. Can we just use `encoding/json` instead?

Reply:
Proto messages are how we generally handle marshaling throughout the SDK, so probably for consistency.

Reply:
Should I switch to `protojson` or is `encoding/json` fine? It seems like both are used throughout the SDK. I personally think `encoding/json` is a simpler solution.

Reply:
Since these `kos` key outputs are not proto.Messages, we won't even be able to use protojson here. encoding/json is fine